helpers/helpers.go

```diff
@@ -194,3 +211,38 @@ func ValidateMinSignatures(n int) error {
 	}
 	return nil
 }
+
+func ValidateKeyLength(key []byte) bool {
+	if len(key) == KeyLengthEd25519 {
+		return true
+	}
+	if len(key) == KeyLengthECDSA {
+		return true
+	}
+	if len(key) == KeyLengthGOST {
+		return true
+	}
+	return false
+}
+
+func ParseBool(text string) bool {
+	const True = "true"
+	return text == True
+}
```

Comment:
https://pkg.go.dev/strconv#ParseBool

Reply:
I removed this function and returned the old behavior.